src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub fn left_pad(s: &str, pad: u32) -> String
 /// pad a string to the left with ``pad`` ``padchar``
 pub fn left_pad_char(s: &str, pad: u32, padchar: char) -> String
 {
-    let mut out = String::new();
+    let mut out = String::with_capacity( s.as_bytes().len() + (pad as usize) );
```

Comment:
I thought about doing that. It happens that your patch is based of 0.1.0 while I already have 0.2.0 that declare ````pad```` as ````usize````. I you can rebase, I'll be happy to merge.